R/class.R

```diff
@@ -74,7 +74,8 @@ validate_messydate <- function(x) {
 
   if (any(grepl("[A-WYZa-z]", values))) {
     stop(
-      "The only alpha character allowed in messy dates is 'X' for unspecified time components",
+      "The only alpha character allowed in messy dates is 'X' for
```

Comment:
Is this `{messydates}` the package name or "messy dates" the concept or `messydt` the object class?